Week-1/styles.css

```diff
@@ -0,0 +1,108 @@
+@import url('https://fonts.googleapis.com/css?family=Lato');
+
+*{
+    font-family: 'Lato';
+}
+
+menu{
+    list-style: none;
+    padding-left: 25px;
+}
+#header{
+    background-color: #3A3B3C;
+    color :White;
+    margin-left: 80px;
+    width: 100%;
+    height: 350px;
+    position:relative;
+
+
+}
+#navigation{
```

Comment:
same here nav tag

Reply:
Resolved the issue, removed the Id Tag from the nav tag and CSS file